src/screens/OtherProfile/index.js

```diff
@@ -149,10 +144,9 @@ const OtherProfile = () => {
         setOtherProfileFeed(clonedFeeds, dispatchOtherProfile);
         StorageUtils.otherProfileFeed.setForKey(otherId, JSON.stringify(clonedFeeds));
       }
-      setLoading(false);
       setPostOffset(Number(result.offset));
     } catch (e) {
-      setLoading(false);
+      console.log('error', e);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `setLoading(false);` might cause an issue if there's a UI component depending on this state to show/hide loading indicators. If `isLoading` from `useOtherProfileScreenHooks` is replacing it, ensure it's being set correctly in the catch block as well.



<!-- This is an auto-generated comment by OSS CodeRabbit -->